exchanges/bitstamp/bitstamp_test.go

```diff
@@ -500,174 +516,165 @@ func TestSubmitOrder(t *testing.T) {
 	t.Parallel()
 
 	if !mockTests {
-		sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 	}
-
-	var orderSubmission = &order.Submit{
+	o, err := b.SubmitOrder(context.Background(), &order.Submit{
 		Exchange: b.Name,
 		Pair: currency.Pair{
 			Base:  currency.BTC,
 			Quote: currency.USD,
 		},
 		Side:      order.Buy,
 		Type:      order.Limit,
-		Price:     1,
-		Amount:    1,
-		ClientID:  "meowOrder",
+		Price:     2211.00,
+		Amount:    45,
+		ClientID:  "123456789",
 		AssetType: asset.Spot,
-	}
-	response, err := b.SubmitOrder(context.Background(), orderSubmission)
-	switch {
-	case sharedtestvalues.AreAPICredentialsSet(b) && (err != nil || response.Status != order.New) && !mockTests:
-		t.Errorf("Order failed to be placed: %v", err)
-	case !sharedtestvalues.AreAPICredentialsSet(b) && err == nil && !mockTests:
-		t.Error("Expecting an error when no keys are set")
-	case mockTests && err == nil:
-		t.Error("Expecting an error until QA pass is completed")
+	})
+	if !mockTests {
+		assert.ErrorContains(t, err, "You have only 0 USD available. Check your account balance for details.")
+	} else {
+		assert.NoError(t, err, "SubmitOrder should not error")
+		assert.Equal(t, float64(45), o.Amount, "Amount should be correct")
+		assert.Equal(t, asset.Item(2), o.AssetType, "AssetType should be correct")
```

Comment:
```suggestion
		assert.Equal(t, asset.Spot, o.AssetType, "AssetType should be correct")
```

As it's less ambiguous

Reply:
Corrected